File: Practice/src/introduction_coding_tests/CharactersThatAppearOnlyOnce.java
package introduction_coding_tests;

import java.util.Arrays;
import java.util.stream.Collectors;

public class CharactersThatAppearOnlyOnce {
//	한 번만 등장한 문자
	public String solution(String s) {
		return Arrays.stream(s.split("")).sorted().filter(i -> s.indexOf(i) == s.lastIndexOf(i))
				.collect(Collectors.joining());
	}
}

File: Practice/src/introduction_coding_tests/CharactersThatAppearOnlyOnce.js
function solution(s) {
    return s
        .split('')
        .filter((v) => s.indexOf(v) == s.lastIndexOf(v))
        .sort()
        .join('');
    // return [...s]
    //     .filter((c) => s.match(new RegExp(c, 'g')).length == 1)
    //     .sort()
    //     .join('');
}
